libs/store-selection/src/variables-interfaces/store-variables.tsx

```diff
@@ -0,0 +1,70 @@
+import { GiFoodTruck } from 'react-icons/gi';
+import { MdFoodBank } from 'react-icons/md';
+import { RiHandbagFill } from 'react-icons/ri';
+
+import { DateType, ServiceType } from './store-interfaces';
+
+export const openType1 = {
```

Comment:
pls, rename variables of openType1, openType2, openType3

Reply:
Done